src/Makefile

```diff
@@ -85,6 +85,25 @@ ifeq ($(USE_JEMALLOC),no)
 	MALLOC=libc
 endif
 
+ifdef SANITIZER
+ifeq ($(SANITIZER),address)
+	CFLAGS+=-fsanitize=address -fno-sanitize-recover=all
```

Comment:
Adding `-fno-omit-frame-pointer` normally helps to get better stack traces in the results, anything you have tried?

Reply:
Actually, I get pretty accurate stacktraces but maybe that's related to compiler version/arch I tried. I added that flag anyway, it might be helpful in another setup.